### ISSUE TITLE
Hold request - Scan request loses "Held" status once the scanning is complete

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#37547]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.45-2
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

## Action Performed:
1. Go to staging.new.expensify.com
2. Create a scan request
3. Go to request details page
4. Click 3-dot menu > Hold request
5. Add a reason and save it
6. Wait for scanning to complete
7. When it is complete. go to details page

## Expected Result:
The "Held" status of the scan request will be preserved

## Actual Result:
The scan request loses "Held" status once the scanning is complete

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/e96caddc-35b6-4701-bd67-3972a58bd636

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by CortneyOfstad** [comment]
I am not able to recreate this via Staging. When the scanning was completed, the request is still on Hold, so I don't see this being a bug:
<img width="985" alt="Screenshot 2024-02-29 at 12 33 54 PM" src="https://github.com/Expensify/App/assets/10926327/83366c95-2aa0-42c8-9650-316ba36942ee">

**Comment by CortneyOfstad** [comment]
Going to close and if the issue pops up again, we can readdress.

**Comment by lanitochka17** [comment]
@CortneyOfstad Hello
Issue is still reproducible on the latest build  1.4.46-1


https://github.com/Expensify/App/assets/78819774/8fccbf31-be62-4f10-904b-1c75ad37f154

**Comment by stephanieelliott** [comment]
Per https://github.com/Expensify/App/issues/37536#issuecomment-1977234505 you shouldn't be able to hold a scanning request. So we'll be getting rid of the hold/scan behavior altogether, no need to fix this one
